bin/pycbc_live

```diff
@@ -1242,6 +1256,37 @@ with ctx:
                         if len(results[ifo][key]):
                             results[ifo][key] = results[ifo][key][idx]
 
+            # Calculate and add the psd variation for the results
+            if args.psd_variation:
+                logging.info('Setting all ifo filters to none')
```

Comment:
Not sure how many logging statements are needed. Thinking maybe just the single statement to say that PSD variation has been calculated for an ifo?